app/features/send/share-cashu-token.tsx

```diff
@@ -96,6 +103,19 @@ export function ShareCashuToken({ token }: Props) {
           </Carousel>
         </div>
       </PageContent>
+      {showOk && (
```

Comment:
is it not a bit unusual to show it only after copied? why not just have it shown all the time?